background/handle-site-access.js

```diff
@@ -0,0 +1,24 @@
+chrome.permissions.contains(
+  {
+    permissions: ["webRequest", "webRequestBlocking", "cookies"],
```

Comment:
Is it necessary to request these permissions? I don't think there's any way to revoke them from the UI.